onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -369,6 +370,7 @@ Status SequentialExecutor::Execute(const SessionState& session_state, const std:
                                                          {"activation_size", std::to_string(input_activation_sizes)},
                                                          {"parameter_size", std::to_string(input_parameter_sizes)},
                                                          {"output_size", std::to_string(total_output_sizes)},
+                                                         {"thread_pool", concurrency::ThreadPool::StopProfiling(session_state.GetThreadPool())},
```

Comment:
"thread_pool" -> "thread_scheduling_stats" [ something like that so it is a phrase describing the data returned ]

Reply:
Done.

---
In reply to: [581166473](https://github.com/microsoft/onnxruntime/pull/6748#discussion_r581166473) [](ancestors = 581166473)